docs/contributor/image_mode.md

```diff
@@ -0,0 +1,217 @@
+# Image Mode for MicroShift
+
+Image mode is a new approach to operating system deployment that lets users build,
+deploy, and manage Red Hat Enterprise Linux as a bootable container (`bootc`) image.
```

Comment:
Should we mentioned that "bootable container images" are based on OCI images so it's explicit that there's nothing new about build process (docker/podman build) or transport (regular docker registries) and it's more about how bootc handles all of it?